style.css

```diff
@@ -1,3 +1,67 @@
 .main {
+  width: 50%;
+  margin: auto;
+}
+
+.main-title {
+  text-align: center;
+}
+
+.dflex {
+  display: flex;
+  width: auto;
+  flex-wrap: wrap;
+  margin: 3rem 0 !important;
+}
+
+.row-line {
+  display: flex !important;
+  justify-content: space-between;
+}
+
+.div1 {
+  display: flex;
+  justify-content: space-between;
+  width: 50%;
+  flex-wrap: wrap;
+  margin-right: 0.5rem;
+}
+
+.break-line {
+  margin: auto;
+  margin-bottom: 1rem;
+  color: black;
+  opacity: 1;
   width: 30%;
 }
+
+.addbook-title {
+  text-align: center;
+  margin: 2rem 0;
+}
+
+.form-flex {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 65%;
+  margin: auto;
+  position: relative;
+  margin-bottom: 5rem;
+}
+
+.form-flex > * {
+  margin-bottom: 1rem;
+}
+
+.form-flex > input {
+  border: 2px solid black;
+}
+
+.addBook {
+  width: 20%;
+  margin: auto;
+  position: absolute;
+  bottom: -2rem;
+  right: 0;
+}
```

Comment:
Hello @Emmyn5600
Please check better the github repository for this review, you checked the repo for https://lfmnovaes.github.io/awesomebooks, ours is https://github.com/ramiaquista/awesome-books
We're going to be waiting for a review from you since this one was incorrect because it was a different repo.
Ours implementation is:
![image](https://user-images.githubusercontent.com/43800831/133477302-0929f6ba-7c0e-452a-8a51-15efa2809ad4.png)

Reply:
@Emmyn5600 We are going to be waiting for an answer! 
Thank's for your time! 😄 

Jose, Ramiro.

Reply:
Hello @ramiaquista you can ask for another review